### PR TITLE
fix: invalid reference to the stack

### PR DESCRIPTION
**Pull request by wellsiau-aws**
Fixes # [309](https://github.com/aws-samples/aws-cdk-intro-workshop/issues/309)

The sample code [here](https://github.com/aws-samples/aws-cdk-intro-workshop/tree/master/code/python/pipelines-workshop) already referring to the right stack, however the workshop instruction is not up to date.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: workshop/content/30-python/70-advanced-topics/200-pipelines/4000-build-stage.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ Create a new file in `cdk_workshop` called `pipeline_stage.py` with the code bel
 from aws_cdk import (
     core
 )
-from pypipworkshop_stack import PypipworkshopStack
+from cdk_workshop_stack import CdkWorkshopStack
```
Comment (corymhall) [inline workshop/content/30-python/70-advanced-topics/200-pipelines/4000-build-stage.md:15]:
```suggestion
from .cdk_workshop_stack import CdkWorkshopStack
```

File: workshop/content/30-python/70-advanced-topics/200-pipelines/5000-test-actions.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ With a slight modification to `cdk_workshop/pipeline_stage.py` we can expose the
 from aws_cdk import (
     core
 )
-from pypipworkshop_stack import PypipworkshopStack
+from cdk_workshop_stack import CdkWorkshopStack
```
Comment (corymhall) [inline workshop/content/30-python/70-advanced-topics/200-pipelines/5000-test-actions.md:139]:
```suggestion
from .cdk_workshop_stack import CdkWorkshopStack
```

**Review by corymhall — CHANGES_REQUESTED** [review]
@wellsiau-aws thanks for the PR! Just a minor fix and then we are good to go.

File: workshop/content/30-python/70-advanced-topics/200-pipelines/4000-build-stage.md (hunk 1 above)
Comment (wellsiau-aws) [reply workshop/content/30-python/70-advanced-topics/200-pipelines/4000-build-stage.md]: thanks, I added the fix here [b3d4f07](https://github.com/aws-samples/aws-cdk-intro-workshop/pull/320/commits/b3d4f073ff6fa29e8da4f583306ee65c6ccae9bf)

File: workshop/content/30-python/70-advanced-topics/200-pipelines/5000-test-actions.md (hunk 2 above)
Comment (wellsiau-aws) [reply workshop/content/30-python/70-advanced-topics/200-pipelines/5000-test-actions.md]: added the fix here [b3d4f07](https://github.com/aws-samples/aws-cdk-intro-workshop/pull/320/commits/b3d4f073ff6fa29e8da4f583306ee65c6ccae9bf)

**Comment by wellsiau-aws** [comment]
New update:
- [b3d4f07](https://github.com/aws-samples/aws-cdk-intro-workshop/pull/320/commits/b3d4f073ff6fa29e8da4f583306ee65c6ccae9bf) : update import to relative path

**Review by corymhall — APPROVED** [review]

**Pull request merged**